### PR TITLE
Slight rewording of "Writing Data to OAM Memory" to clarify DMA instructions

### PR DESCRIPTION
**Pull request by TheRektafire**

### CODE REVIEW

**Review by avivace — APPROVED** [review]

File: src/OAM.md [hunk 1]
```diff
@@ -92,9 +92,7 @@ lower-priority sprite's "BG and Window over OBJ" attribute.
 
 ## Writing Data to OAM Memory
 
-The recommended method is to write the data to normal RAM first, and to
-copy that RAM to OAM by using the DMA transfer function, initiated
-through DMA register (FF46). Besides, it is also possible to
-write data directly to the OAM area by using normal LD instructions, but this
+The recommended method is to write the data to normal RAM first, then copy that RAM to OAM by using the DMA transfer function, initiated by writing the high byte of the RAM page on which the data is stored to the DMA register ($FF46). While it is also possible to
```
Comment (ISSOtm) [inline src/OAM.md:95]: Clarifying that "normal RAM" means WRAM or SRAM may be better.

Comment (ISSOtm) [inline src/OAM.md:95]: We don't usually talk about "RAM page", so this term must not be used.
Suggested: "... by writing the high byte of the address, which must be a multiple of $100, to..."

Comment (ISSOtm) [inline src/OAM.md:95]: Please break the line at the end of the sentence, i.e. before the "While".

**Review by ISSOtm — CHANGES_REQUESTED** [review]

**Comment by ISSOtm** [comment]
No activity for a while, should we apply the modifications ourselves?

**Comment by ISSOtm** [comment]
PR seems inactive, I'll be applying the changes myself and merging.

**Review by ISSOtm — APPROVED** [review]

**Pull request merged**